piclassifier/cptvtest.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python3
+import argparse
+
+import socket
+import time
+import os
+from cptv import CPTVReader
+import sys
+import numpy as np
+import h5py
+
+SOCKET_NAME = "/var/run/lepton-frames"
+test_cptv = "test.cptv"
+test_h5py = "/home/zaza/Cacophony/classifier-pipeline/pithermal65.h5py"
```

Comment:
These appear to be unused (and we don't want paths that only make sense on your machine anyway)